docs/intro/getting-started-with-sphinx.rst

```diff
@@ -0,0 +1,135 @@
+Getting Started with Sphinx
+===========================
+
+Sphinx is a powerful documentation generator that
+has many great features for writing technical documentation including:
+
+* Generate web pages, printable PDFs, documents for e-readers (ePub),
+  and more all from the same sources
+* You can use reStructuredText or :ref:`Markdown <intro/getting-started-with-sphinx:Using Markdown with Sphinx>`
+  to write documentation
+* An extensive system of cross-referencing code and documentation
+* Syntax highlighted code samples
+* A vibrant ecosystem of first and third-party extensions_
+
+.. _extensions: http://www.sphinx-doc.org/en/master/ext/builtins.html#builtin-sphinx-extensions
+
+.. note::
+
+    Sphinx is a great choice for building technical documentation
+    and many projects large and small use Sphinx
+    including the Django project and the Python programming language.
+    However, Read the Docs also supports :doc:`MkDocs </intro/getting-started-with-mkdocs>`,
+    another tool for writing and building documentation.
```

Comment:
I don't think we necessarily want to point out mkdocs here, if anything because we're not giving a reason for the user to consider mkdocs with this sentence. I think our position is that the author would be better off with Sphinx and RTD anyways, due to the lack of feature parity on mkdocs. I'd be :+1: on dropping the mkdocs callout here.

Reply:
I removed it. We could also discuss some of the differences like lack of PDF/e-reader support in mkdocs but I don't know if it is necessary.